File: VB/04_Shapes/ReorderOverlappingShapes.vb
Imports Spire.Presentation

Namespace ReorderOverlappingShapes
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim ppt As New Presentation()
			'Load file
			ppt.LoadFromFile("..\..\..\..\..\..\Data\OverlappingShapes.pptx")

			'Get the first shape of the first slide
			Dim shape As IShape = ppt.Slides(0).Shapes(0)
			'Change the shape's zorder
			ppt.Slides(0).Shapes.ZOrder(1, shape)

			'Save the document
			Dim result As String = "ReorderOverlappingShapes.pptx"
			ppt.SaveToFile(result, FileFormat.Pptx2013)
			PresentationDocViewer(result)
		End Sub

		Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/04_Shapes/ReorderOverlappingShapes.cs
using System;
using System.Windows.Forms;
using Spire.Presentation;

namespace ReorderOverlappingShapes
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation ppt = new Presentation();
            //Load file
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\OverlappingShapes.pptx");

            //Get the first shape of the first slide
            IShape shape = ppt.Slides[0].Shapes[0];
            //Change the shape's zorder
            ppt.Slides[0].Shapes.ZOrder(1, shape);

            //Save the document
            string result = "ReorderOverlappingShapes.pptx";
            ppt.SaveToFile(result, FileFormat.Pptx2013);
            PresentationDocViewer(result);
		}
	
		private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}